persistence/schema/gov.go

```diff
@@ -3,440 +3,151 @@ package schema
+		'%s',
+		'%s',
+		'%s'
+		)`,
+		ValTypeString,
+		ValTypeBigInt,
+		ValTypeSmallInt,
+	)
 )
 
 func InsertParams(params *genesis.Params) string {
-	return fmt.Sprintf(`INSERT INTO %s VALUES(%d, '%s', %d, %d, %d, %d, %d, %d,
-						'%s',%d,%d,%d,%d,%d,
-						'%s',%d,%d,%d,%d,
-						'%s',%d,%d,%d,%d,
-						%d,%d,%d,%d,
-						'%s','%s','%s','%s','%s','%s','%s','%s','%s','%s',
-						'%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s',
-						'%s','%s','%s','%s','%s','%s','%s','%s','%s',
-						'%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s',
-						'%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s','%s',%d)`,
-		ParamsTableName,
-		params.BlocksPerSession,
-		params.AppMinimumStake,
-		params.AppMaxChains,
-		params.AppBaselineStakeRate,
-		params.AppStakingAdjustment,
-		params.AppUnstakingBlocks,
-		params.AppMinimumPauseBlocks,
-		params.AppMaxPauseBlocks,
-
-		params.ServiceNodeMinimumStake,
-		params.ServiceNodeMaxChains,
-		params.ServiceNodeUnstakingBlocks,
-		params.ServiceNodeMinimumPauseBlocks,
-		params.ServiceNodeMaxPauseBlocks,
-		params.ServiceNodesPerSession,
-
-		params.FishermanMinimumStake,
-		params.FishermanMaxChains,
-		params.FishermanUnstakingBlocks,
-		params.FishermanMinimumPauseBlocks,
-		params.FishermanMaxPauseBlocks,
-
-		params.ValidatorMinimumStake,
-		params.ValidatorUnstakingBlocks,
-		params.ValidatorMinimumPauseBlocks,
-		params.ValidatorMaxPauseBlocks,
-		params.ValidatorMaximumMissedBlocks,
-
-		params.ValidatorMaxEvidenceAgeInBlocks,
-		params.ProposerPercentageOfFees,
-		params.MissedBlocksBurnPercentage,
-		params.DoubleSignBurnPercentage,
+	val := reflect.ValueOf(params)
+	var subQuery string
+	for _, k := range govParamMetadataKeys {
+		pnt := govParamMetadataMap[k]
+		pVal := val.Elem().FieldByName(pnt.PropertyName)
+
+		subQuery += `(`
+		switch govParamMetadataMap[k].PropertyType {
+		case ValTypeString:
+			var stringVal string
+			switch vt := pVal.Interface().(type) {
+			case []byte:
+				stringVal = hex.EncodeToString(vt)
+			case string:
+				stringVal = vt
+			}
+			subQuery += fmt.Sprintf("'%s', %d, true, '%s', '%s'", k, DefaultBigInt, pnt.PropertyType, stringVal)
+
+		case ValTypeSmallInt, ValTypeBigInt:
+			subQuery += fmt.Sprintf("'%s', %d, true, '%s', %d", k, DefaultBigInt, pnt.PropertyType, pVal.Interface())
+		}
+		subQuery += `),`
+	}
+	return fmt.Sprintf(`INSERT INTO %s VALUES %s`, ParamsTableName, subQuery[:len(subQuery)-1])
+}
 
-		params.MessageDoubleSignFee,
-		params.MessageSendFee,
-		params.MessageStakeFishermanFee,
-		params.MessageEditStakeFishermanFee,
-		params.MessageUnstakeFishermanFee,
-		params.MessagePauseFishermanFee,
-		params.MessageUnpauseFishermanFee,
-		params.MessageFishermanPauseServiceNodeFee,
-		params.MessageTestScoreFee,
-		params.MessageProveTestScoreFee,
+func GetParamQuery(paramName string) string {
```

Comment:
This will need  to take in a `height` param and we may need to modify the entire persistence module interface to take in a height value as well.

Regarding the query:
1. Remove enabled=true
2. Use the passed in height

cc @andrewnguyen22 to confirm